pkg/trace/test/agent.go

```diff
@@ -29,40 +29,39 @@ type agentRunner struct {
 	mu  sync.RWMutex // guards pid
 	pid int          // agent pid, if running
 
-	port    int         // agent port
-	log     *safeBuffer // agent log
-	ddAddr  string      // Datadog API address (host:port)
+	port    int         // agent receiver port
+	log     *safeBuffer // agent log output
+	ddAddr  string      // Datadog intake address (host:port)
+	binpath string      // full trace-agent binary path
 	verbose bool
 }
 
 func newAgentRunner(ddAddr string, verbose bool) (*agentRunner, error) {
-	if _, err := exec.LookPath("trace-agent"); err != nil {
-		// trace-agent not in $PATH, try to install
+	binpath := filepath.Join(os.TempDir(), "trace-agent-testbin")
```

Comment:
We could maybe use ioutil.TempDir or ioutil.TempFile to isolate the tests from possible concurrent runs?

Reply:
Yeah that's probably a better idea. I'll make that change.